### PR TITLE
Use app version to generate scss filename

### PR DESCRIPTION
**Pull request by skjnldsv**
To force our scss cache to clear

### CODE REVIEW

File: lib/private/Template/SCSSCacher.php [hunk 1]
```diff
@@ -138,7 +138,8 @@ public function process(string $root, string $file, string $app): bool {
 	 */
 	public function getCachedCSS(string $appName, string $fileName): ISimpleFile {
 		$folder = $this->appData->getFolder($appName);
-		return $folder->getFile($this->prependBaseurlPrefix($fileName));
+		$cachedFileName = $this->prependVersionPrefix($this->prependBaseurlPrefix($fileName), $appName);
```
Comment (juliushaertl) [inline lib/private/Template/SCSSCacher.php]: Looks like `$folder->getFile()` has been lost

File: lib/private/Template/SCSSCacher.php [hunk 2]
```diff
@@ -322,19 +323,34 @@ private function rebaseUrls(string $css, string $webDir): string {
...
+		return substr(md5($this->urlGenerator->getBaseUrl() . $frontendController), 0, 4) . '-' . $cssFile;
+	}
+
+	/**
+	 * Prepend hashed app version hash
+	 * @param string $cssFile
+	 * @param string $appId
+	 * @return string
+	 */
+	private function prependVersionPrefix(string $cssFile, string $appId): string {
```
Comment (juliushaertl) [inline lib/private/Template/SCSSCacher.php]: Any specific reason to not just have one prependPrefix method?

**Review by juliusknorr — DISMISSED** [review]

File: lib/private/Template/SCSSCacher.php (hunk 2 above)
Comment (skjnldsv) [reply lib/private/Template/SCSSCacher.php]: Simplicity? :p

File: lib/private/Template/SCSSCacher.php (hunk 1 above)
Comment (skjnldsv) [reply lib/private/Template/SCSSCacher.php]: Nice catch. Somehow missed that! 🙈

File: lib/private/Template/SCSSCacher.php (hunk 2 above)
Comment (juliushaertl) [reply lib/private/Template/SCSSCacher.php]: Ok, don't mind that much :wink:

**Review by juliusknorr — APPROVED** [review]
Looks good and works as expected.

**Review by rullzer — APPROVED** [review]

**Pull request merged**